src/documentation/sdks/nexmark.md

```diff
@@ -175,9 +175,9 @@ Available suites are:
     --workerMachineType=n1-highmem-8 \
     --stagingLocation=<a gs path for staging> \
     --runner=DataflowRunner \
-    --tempLocation=gs://talend-imejia/nexmark/temp/ \
-    --stagingLocation=gs://talend-imejia/nexmark/temp/staging/ \
-    --filesToStage=target/beam-sdks-java-nexmark-2.1.0-SNAPSHOT.jar
```

Comment:
Better add {{ site.release_latest }} instead of -[version]  This applies to the other uses of version too.

Reply:
yes, totally agree, good suggestion !